build.sbt

```diff
@@ -153,8 +166,10 @@ lazy val generic = (project in file("generic"))
     libraryDependencies ++=
       COMPILE(
         "com.chuusai" %% "shapeless" % shapelessVersion
-      )
+      ),
+    Test / scalacOptions += "-Wconf:cat=other-pure-statement&msg=org.scalatest.Assertion:s"
```

Comment:
This should be on the common settings since we want this exception on all tests.

Reply:
I moved it to commonSettings.